### PR TITLE
Update back-end-server.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3690

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 1]
```diff
@@ -116,10 +122,10 @@ Skype for Business Server supports AlwaysOn with the following database software
 - SQL Server 2012 SP2 and CU2 Enterprise Edition
 
 > [!NOTE]
-> SQL Server 2017 and SQL Server 2016 are the only versions supported by Skype for Business Server 2019.
+> SQL Server 2019, 2017 and 2016 are the only versions supported by Skype for Business Server 2019.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion
> SQL Server 2019, 2017, and 2016 are the only versions supported by Skype for Business Server 2019.
```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 2]
```diff
@@ -116,10 +122,10 @@ Skype for Business Server supports AlwaysOn with the following database software
...
 
 > [!NOTE]
-> Always On Availability Groups is **not** supported in SQL 2016 and 2017 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
+> Always On Availability Groups is **not** supported in SQL 2016, 2017 and 2019 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion
> Always On Availability Groups is **not** supported in SQL 2016, 2017, and 2019 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 3]
```diff
@@ -141,7 +149,7 @@ Skype for Business Server supports SQL Server failover clustering with the follo
 To use SQL failover clustering, you should first set up and configure the SQL Server cluster before deploying your Front End pool. For best practices and setup instructions for failover clustering in SQL Server 2012, see [https://technet.microsoft.com/en-us/library/hh231721.aspx](https://technet.microsoft.com/en-us/library/hh231721.aspx).
 
 > [!NOTE]
-> SQL Server 2017 and SQL Server 2016 are the only versions supported by Skype for Business Server 2019.
+> SQL Server 2019, 2017 and SQL Server 2016 are the only versions supported by Skype for Business Server 2019.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion
> SQL Server 2019, 2017, and SQL Server 2016 are the only versions supported by Skype for Business Server 2019.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**